assessment/assessment.go

```diff
@@ -88,7 +88,7 @@ func (inst *Instance) GetWikiRecs(Properties []string) schematree.PropertyRecomm
 
 	var res *http.Response
 	var err error
-	for true { // retry like a maniac
+	for { // retry like a maniac
```

Comment:
maniac is not a good choice of words

Reply:
Fixing in the branch. I made the pull request to test the CI. I will close this one now and create a new one. that should also trigger gosec.

Reply:
Apparently, pushing the change triggers gosec as well. Keeping this pull request open. We will merge it once we have also the remaining linting/gosec issues fixed.

Reply:
fixed in ab9661b